src/Classes/Item.lua

```diff
@@ -283,28 +285,37 @@ function ItemClass:ParseRaw(raw)
 					importedLevelReq = tonumber(specVal)
 				elseif specName == "LevelReq" then
 					self.requirements.level = tonumber(specVal)
-				elseif specName == "Has Alt Variant" then
-					self.hasAltVariant = true
-				elseif specName == "Has Alt Variant Two" then
-					self.hasAltVariant2 = true
-				elseif specName == "Has Alt Variant Three" then
-					self.hasAltVariant3 = true
-				elseif specName == "Has Alt Variant Four" then
-					self.hasAltVariant4 = true
-				elseif specName == "Has Alt Variant Five" then
-					self.hasAltVariant5 = true
+				elseif specName:match("Has Alt Variant") then
+					local variantName = specName == "Has Alt Variant" and "One" or specName:gsub("Has Alt Variant ","")
+					self.variantList = self.variantList or { Base = { } }
+					self.variantList[variantName] = {}
+					if specVal:match("%(%d+,%d+%)") then
+						local variantRange1, variantRange2
+						for i in specVal:gmatch("%d+") do
+							if variantRange1 then
+								variantRange2 = tonumber(i)
+							else
+								variantRange1 = tonumber(i)
+							end
+						end
```

Comment:
```suggestion
						local variantRange1, variantRange2 = line:match("%((%d+),(%d+)%)")
```
This should work as well afaik

Reply:
I had this originally but ran into some issues, do not remember what they were but i can take a look at setting it back to this